### DISCUSSION TITLE
Is this support Expo project ?

### DISCUSSION THREAD

**Discussion by tuanngocptn** [baronha/ting#26] [Q&A] [answered]
Hello.

I'm starting to set up my new project and I'm loving to use `ting` for my project. But I concern Is this support Expo project?. Because I will using  React Native Expo for my project.

Thanks

**Comment by baronha** [comment, marked as answer]
Please follow this issue: https://github.com/baronha/ting/issues/19

Thank @tuanngocptn

**Reply by tuanngocptn** [reply]
Thank you so much
